ISSUE-TEMPLATE.md

```diff
@@ -0,0 +1,18 @@
+# Issue name
+
+* Related Issues : _#IssueNumber if necessary_
+
+## Introduction
+
+A short description of what the features or the bugs are. Try to keep it simple and within a
+single-paragraph.
+
+## Issue Description
+
+Describe the bug found or the feature that you are proposing
+
+If they are part of some common pattern hard to express, show
+how one can currently get a similar effect and describe its
+drawbacks. 
```

Comment:
I have no idea what this means. This paragraph isn't very clear to me. What is a "common pattern hard to express"? By "similar effect" do you mean how to reproduce the bug? Or is it something related with new features?